src/main/java/frc/robot/subsystems/IOdometryPose.java

```diff
@@ -0,0 +1,19 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.math.Matrix;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Transform2d;
+import edu.wpi.first.math.kinematics.SwerveDriveWheelPositions;
+import edu.wpi.first.math.numbers.N1;
+import edu.wpi.first.math.numbers.N3;
+import frc.robot.poseestimator.OdometryObservation;
+
+public interface IOdometryPose {
```

Comment:
i would call it OdomertyCalculator

Reply:
I wouldn't

Reply:
The interface does not represent OdometryPose, it represents a class that calculates odometry with inputs and outputs. Find a better name for the class if not OdomertyCalculator,